.github/workflows/ci.yml

```diff
@@ -166,6 +169,9 @@ jobs:
           TAG: ${{ github.event.ref }}
           OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
           OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
+          #do not remove signing key and password or signatures will not be published
+          ORG_GRADLE_PROJECT_signingKey: ${{ secrets.ARTIFACT_SIGNING_KEY }}
```

Comment:
so having that prefix makes it to build step?

https://github.com/plantuml/plantuml/blob/3c53361064052a3174cf8090c464cb6de60aafd5/build.gradle.kts#L163-L166

Reply:
Could you please elaborate on that a little further ? I did not understand your question.

But yes this is related to L163 conditions

Reply:
I was asking if ORG_GRADLE_PROJECT_* prefix allows value to be visible in above `signing` section

Reply:
Gradle project properties is realy powerfull.
One way to pass a project property is using enviroment variable.
`ORG_GRADLE_PROJECT_`is a standard prefix. 

Example: if you have a project property named `myProp`, instead of adding `myProp=some_value`in `gradle.properties`you can use an environment variable `ORG_GRADLE_PROJECT_myProp=some_value`

See this part of the documentation : 
https://docs.gradle.org/current/userguide/build_environment.html#sec:project_properties